docker-wrapper.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Variables setting
+HOST=`hostname -i`
+petalscli="/opt/petals-deployer/bin/petals-cli.sh -h $ENV_CLI_HOST -n $ENV_CLI_PORT -u $ENV_CLI_USER -p $ENV_CLI_PASS -y -c "
+
+# Informational dump
+echo "############################ PETALS DEPLOYER ############################"
+echo "###     Starting deploy script with following parameters:" 
+echo " - petals host: $ENV_CLI_HOST" && echo " - petals port: $ENV_CLI_PORT" && echo " - petals user: $ENV_CLI_USER" && echo " - petals pass: $ENV_CLI_PASS"
+echo " - deploy file: $ENV_DEPLOY_FILE"
+echo " - URLS to deploy: $ENV_DEPLOY_URLS"
+echo " - CLI host: ${HOST}"
+echo " - CLI version :"
+/opt/petals-deployer/bin/petals-cli.sh --version
+
+# CLI environment file setting
+echo ""
+echo "###     Setting configuration..." 
+# For now this deployer must be used without "--net=host" option
```

Comment:
Remove this comment.
--net options is meaningful with Docker, but not with Kubernetes. So, from now, --net is banned. :smiling_imp: